portfolio/src/main/webapp/background.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title> Carol's Portfolio </title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"> </script>
+  </head>
+  <body>
+    <div class="container">
+      <div class="menu">
+        <ul>
+            <li class="logo"> <a href="index.html"> <img src="images/logo.jpg"> </a> </li>
+            <li> <a href="index.html"> About Me </a> </li>
+            <li class="currentLink"> <a href="background.html" style="color:white"> Background </a> </li>
+            <li> <a href="photos.html"> Photos </a> </li>
+            <li> <a href="contact.html"> Contact Me </a> </li>
+        </ul>
+      </div>
+    </div>
+    <h1> Background </h1>
+    <p> I am a computer science major at Caltech, expecting to graduate in June 2022. <br> 
```

Comment:
This would look better if `<p>` is alone on the line. That would also make it consistent with the fact that `</p>` is alone on its line.